kbase.yml

```diff
@@ -2,16 +2,16 @@ module-name:
     dRep
 
 module-description:
-    A KBase module
+    Dereplicate genomes using ANI
 
 service-language:
     python
 
 module-version:
-    0.0.1
+    1.0.0
 
 data-version:
-    0.0.1
+    0.0.3
 
 owners:
     [suminw]
```

Comment:
Can't comment on the file itself, but need to remove `lib/dRep/.dRepImpl.py.swp`

Reply:
I would globally gitignore *.swp files

Reply:
Also need to remove and gitignore .pyc files, such as `lib/dRep/__pycache__/dRepImpl.cpython-37.pyc`